### PR TITLE
Cisco ASA parsing rules - iso8601

### PR DESCRIPTION
**Pull request by guytamir10**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

### CODE REVIEW

File: Packs/CiscoASA/README.md [hunk 1]
```diff
@@ -12,3 +12,15 @@ You can configure the specific vendor and product for this instance.
 3. When configuring the Syslog Collector, set:
    - vendor as vendor<- Cisco
    - product as product<- ASA
+
+### config timeformat on Cisco ASA
```
Comment (ShirleyDenkberg) [inline Packs/CiscoASA/README.md:16]:
```suggestion
### Configure Timestamp on Cisco ASA
```

File: Packs/CiscoASA/README.md [hunk 2]
```diff
@@ -12,3 +12,15 @@ You can configure the specific vendor and product for this instance.
...
+In Cisco ASA product we support only iso8601 date format with timezone.
```
Comment (ShirleyDenkberg) [inline Packs/CiscoASA/README.md:17]:
```suggestion
In Cisco ASA product we support only ISO8601 date format with the time zone.
```

File: Packs/CiscoASA/README.md [hunk 3]
```diff
@@ -12,3 +12,15 @@ You can configure the specific vendor and product for this instance.
...
+an example ofr the timestamp: "2023-04-09T16:30:00-07:00"
+
+1. Access the Cisco ASA command-line interface (CLI) using a console connection or SSH session.
```
Comment (ShirleyDenkberg) [inline Packs/CiscoASA/README.md:20]:
```suggestion
1. Access the Cisco ASA CLI using a console connection or SSH session.
```

File: Packs/CiscoASA/README.md [hunk 4]
```diff
@@ -12,3 +12,15 @@ You can configure the specific vendor and product for this instance.
...
+2. Enter privileged EXEC mode by typing ```enable``` and entering the enable password.
+3. Enter global configuration mode by typing ```conf t```.
+4. Enter the logging timestamp command to update the timestamp format to iso8601 by typeing
```
Comment (ShirleyDenkberg) [inline Packs/CiscoASA/README.md:23]:
```suggestion
4. Enter the logging timestamp command to update the timestamp format to ISO8601 by typing
```

File: Packs/CiscoASA/ReleaseNotes/1_0_34.md [hunk 5]
```diff
@@ -0,0 +1,3 @@
+#### Parsing Rules
+##### CiscoASA Parsing Rule
+- Update Parsing Rules to support only the iso8601 datetime format.
```
Comment (ShirleyDenkberg) [inline Packs/CiscoASA/ReleaseNotes/1_0_34.md:3]:
```suggestion
Updated the parsing rules to support only the ISO8601 datetime format.
```

**Comment by ShirleyDenkberg** [comment]
@evisochek Doc review completed.

File: Packs/CiscoASA/README.md [hunk 6]
```diff
@@ -12,3 +12,13 @@ You can configure the specific vendor and product for this instance.
 3. When configuring the Syslog Collector, set:
    - vendor as vendor<- Cisco
    - product as product<- ASA
+
+### config timeformat on Cisco ASA
+The only supported date format is RFC 5424, an example: "2023-04-09T16:30:00Z"
```
Comment (evisochek) [inline Packs/CiscoASA/README.md:17]: Please add a comment that if the user will use different timestamp formant time extraction and mapping will not be supported.

Comment (ShirleyDenkberg) [reply Packs/CiscoASA/README.md:17]: Add the following:
Note: If a different timestamp format is used, time extraction and mapping will not be supported.

**Review by evisochek — APPROVED** [review]

**Pull request merged**